packages/at_secondary_server/lib/src/connection/outbound/outbound_client.dart

```diff
@@ -301,18 +310,30 @@ class OutboundClient {
   Future<String?> plookUp(String key) async {
     var result = await lookUp(key, handshake: false);
     lastUsed = DateTime.now();
+    logger.finer('plookup result of the $key: $result');
     return result;
   }
 
   void close() {
     if (outboundConnection != null) {
       outboundConnection!.close();
+      logger.finer('Outbound connection closed');
     }
   }
 
   bool isInValid() {
-    return inboundConnection.isInValid() ||
-        (outboundConnection != null && outboundConnection!.isInValid());
+    bool isInvalid = false;
+    if (inboundConnection.isInValid()) {
+      logger.severe('Outbound Connection to $toAtSign is invalid');
```

Comment:
This does not seem to warrant a SEVERE level of logging?

Also, the logged message is confusing - the message refers to 'Outbound Connection' but the check on the previous line is for inboundConnection

Reply:
I have brought the logger level down as part of https://github.com/atsign-foundation/at_server/pull/1367/commits/f97e5e612ee1c2c211822c18c11f2c9398a226b7. The mismatch of logger messages to inbound/outbound connection name has been fixed in [3f36a53](https://github.com/atsign-foundation/at_server/pull/1367/commits/3f36a53d2a44ab80a2ce76e822e77844dba4d5b6)